lib/resources/majors.tsx

```diff
@@ -5164,6 +5164,201 @@ const bsNeuro: Major = {
+            'EN.661.380 Decision Analytics <br/>',
+          criteria:
+            'AS.230.205[C]^OR^AS.280.345[C]^OR^AS.280.345[C]^OR^EN.553.111[C]^OR^EN.553.112[C]^OR^EN.661.380[C]',
+        },
+
+        {
+          required_credits: 3,
+          description:
+            '<b>Option Two: One Intermediate/Calculus-Based Course (must have Calculus as a prerequisite)</b><br />' +
+            'EN.553.211 Probability and Statistics for the Life Sciences' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.553.310 Probability & Statistics for the Physical Sciences & Engineering' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.553.311 Probability and Statistics for the Biological Sciences and Engineering' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.553.430 Introduction to Statistics' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.560.240 Uncertainty, Reliability and Decision-making',
+          criteria:
+            'EN.553.211[C]^OR^EN.553.310[C]^OR^EN.553.311[C]^OR^EN.553.430[C]^OR^EN.560.240[C]',
+        },
+      ],
+    },
+    {
+      name: 'Entrepreneurship and Management Fundamentals',
+      required_credits: 9,
+      min_credits_per_course: 3,
+      description:
+        'Students must complete three fundamental courses in entrepreneurship and management.',
+      criteria:
+        'EN.660.105[C]^OR^EN.660.250[C]^OR^EN.660.203[C]^OR^EN.661.110[C]^OR^EN.661.250[C]',
+      fine_requirements: [
+        {
+          required_credits: 3,
+          description:
+            '<b>EN.660.105</b> <br />Foundations of American Enterprise**',
+          criteria: 'EN.660.105[C]',
+        },
+        {
+          required_credits: 3,
+          description:
+            '<b>EN.660.250</b> <br />Identifying and Capturing Markets**',
+          criteria: 'EN.660.250[C]',
+        },
+        {
+          required_credits: 3,
+          description: '<b>EN.660.203</b> <br />Financial Accounting**',
+          criteria: 'EN.660.203[C]',
+        },
+        {
+          required_credits: 3,
+          description:
+            '<b>EN.661.110</b> <br />Professional Writing and Communication**',
+          criteria: 'EN.661.110[C]',
+        },
+        {
+          required_credits: 3,
+          description: '<b>EN.661.250</b> <br />Oral Presentations**',
+          criteria: 'EN.661.250[C]',
+        },
+      ],
+    },
+    {
+      name: 'Upper-Level Elective Courses in Entrepreneurship and Management',
+      required_credits: 6,
+      min_credits_per_course: 3,
+      description:
+        'Students must complete two upper-level courses in entrepreneurship and management. ' +
+        'Courses may be chosen from the list below. Students can choose 1 course from the Marketing' +
+        ' & Communications electives listed in a separate table below.',
+      criteria:
+        'AS.180.263[C]^OR^EN.570.334[C]^OR^EN.661.301[C]^OR^EN.661.315[C]^EN.661.317[C]^' +
```

Comment:
should 300, 303, 308, 310, 329, etc etc be included here as well?

Reply:
yeah good catch